docs/user-manuals/uniteddeployment.md

```diff
@@ -83,6 +83,86 @@ spec:
 ...
 ```
 
+## Customize pod configuration of subset
+**FEATURE STATE:** Kruise v1.5.0
+
+In the previous version, we only supported different subsets to define different nodeSelectorTerm, tolerations, replicas.
+In order to meet the needs of different business scenarios, since v1.5.0 we have been patching more fields into the pod configuration, such as: env, resources, image, etc.
```

Comment:
in the manual, it is not necessary to introduce the history of unitedeployment feature. Just simplify the description as follows:

Since  v1.5.0, one can customize pod spec field other than nodeSelectorTerm and tolerations, e.g. env, resources. 

Note: it is not recommended to customize subset image since it may cause chaos into  update function , so plz do not list the image as the customization example